src/index.html

```diff
@@ -10,5 +10,31 @@
   <body>
     <div id="main"></div>
     <script src="./js/index.js"></script>
+    <ul>
+      <button background-color: transparent; style="width:200px;height:96px" ;>
+        HOME
+      </button>
+      <button
+        style="z-index: 50; height: 40px; min-height: 18px; width: 302px"
+        ;
+      >
+        DOWNLOAD MY FULL RESUME
+      </button>
+    </ul>
+    <h1 style="text-align:center;">John Roberts</h1>
+    <h2 position="right" , style="text-align:center;">BUSINESS EXPERT</h2>
```

Comment:
`position="right"` no such style statement on CSS. 
CSS position attributes: https://developer.mozilla.org/en-US/docs/Web/CSS/position

Reply:
Will learn it, thank you